app/modules/common/checkpost.js

```diff
@@ -0,0 +1,21 @@
+const mysql = require(__base + "/app/modules/common/mysql");
```

Comment:
doesnt make sense to put this file inside common/ folder.

there is "post" folder inside /modules - create a clap.js file inside and put validation, check, and post function inside that file